x-pack/test/upgrade_assistant_integration/upgrade_assistant/reindexing.js

```diff
@@ -83,6 +83,29 @@ export default function ({ getService }) {
       });
     });
 
+    it('can resume after reindexing was stopped right after creating the new index', async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/upgrade_assistant/reindex');
+
+      await es.indices.create({ index: 'reindexed-v7-dummydata' });
```

Comment:
Setting the SO manually was quite a challenge and couldn't get it to work (was complaining about a document conflict between versions). I ended up going for this approach of just creating the new index manually before running the reindexing process and was able to mimic the same behaviour.

Reply:
Maybe worth adding a comment indicating this is representative of the new, soon-to-be reindexed index.